app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,28 @@
 class PostsController < ApplicationController
+  def new
+    post = Post.new
+    respond_to do |format|
+      format.html { render :new, locals: { post: } }
+    end
+  end
+
+  def create
+    user = User.find(params[:user_id])
+    post_values = params.require(:post).permit(:title, :text)
+    @new_post = Post.new(author: user, title: post_values[:title], text: post_values[:text])

```

Comment:
- [OPTIONAL] Let's make it a good practice to use strong parameters to better handle params according to [this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)
```ruby
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```